control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -608,12 +608,27 @@ func rebuildCMFromStatusAnnotations(br *eventing.Broker) *corev1.ConfigMap {
 }
 
 func (r *Reconciler) reconcilerBrokerResource(ctx context.Context, topic string, broker *eventing.Broker, secret *corev1.Secret, config *kafka.TopicConfig) (*contract.Resource, error) {
+	brokerKind := broker.Kind
+	if brokerKind == "" {
+		brokerKind = "Broker"
+	}
+	brokerAPIVersion := broker.APIVersion
+	if brokerAPIVersion == "" {
+		brokerAPIVersion = "eventing.knative.dev/v1"
+	}
 	resource := &contract.Resource{
 		Uid:    string(broker.UID),
 		Topics: []string{topic},
 		Ingress: &contract.Ingress{
 			Path:                       receiver.PathFromObject(broker),
 			EnableAutoCreateEventTypes: feature.FromContext(ctx).IsEnabled(feature.EvenTypeAutoCreate),
+			EventTypeOwnerReference: &contract.Reference{
+				Uuid:      string(broker.UID),
+				Name:      broker.Name,
+				Namespace: broker.Namespace,
+				Kind:      brokerKind,
+				Version:   brokerAPIVersion,
+			},
```

Comment:
I would simplify this to something like:
```suggestion
	resource := &contract.Resource{
		Uid:    string(broker.UID),
		Topics: []string{topic},
		Ingress: &contract.Ingress{
			Path:                       receiver.PathFromObject(broker),
			EnableAutoCreateEventTypes: feature.FromContext(ctx).IsEnabled(feature.EvenTypeAutoCreate),
			EventTypeOwnerReference: &contract.Reference{
				Uuid:      string(broker.UID),
				Name:      broker.Name,
				Namespace: broker.Namespace,
				Kind:      "Broker",
				Version:   eventing.SchemeGroupVersion.String(),
			},
```